plugins/backend/catalog-backend-module-okta/config.d.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  integrations?: {
+    /**
+     * Okta configuration
+     */
+    okta?: AccountConfig[];
+  };
+}
```

Comment:
I think the configuration of the provider should be under the catalog.providers key. In this case probably stg like: 
```suggestion
export interface Config {
 catalog?: {
    providers?: {
      /**
     * Okta configuration
     */
      okta?: AccountConfig[];
    }
  };
}
```

Reply:
dunno about the providers part.. what if I want to use the okta credentials somewhere else outside of the catalog.. I think that is why we use the integrations part.. but Im happy either way.

Reply:
ah, you are correct, I thought at first this accountconfig is provider specific configuration, but now I can see it is responsible to configure the okta creds and such things. So let's leave it as is.

Reply:
I already changed it to catalog.providers. No harm I think. We can move it if we need later.